src/kOS/Binding/FlightControlManager.cs

```diff
@@ -270,13 +281,64 @@ public void SelectAutopilotMode(string autopilotMode)
                         SelectAutopilotMode(VesselAutopilot.AutopilotMode.StabilityAssist);
                         break;
                     default:
-                        // If the mode is not recognised, thrown an exception rather than continuing or using a default setting
+                        // If the mode is not recognised, throw an exception rather than continuing or using a default setting
                         throw new KOSException(
                             string.Format("kOS does not recognize the SAS mode setting of {0}", autopilotMode));
                 }
             }
         }
 
+  
+        public FlightGlobals.SpeedDisplayModes getNavMode()
+        {
+            if (Shared.Vessel != FlightGlobals.ActiveVessel)
+            {
+                throw new kOS.Safe.Exceptions.KOSSituationallyInvalidException("NAVMODE can only be accessed for the Active Vessel");
+            }
+            return FlightGlobals.speedDisplayMode;
+        }   
+
+        public void SetNavMode(FlightGlobals.SpeedDisplayModes navMode)
+        {
+            FlightGlobals.SetSpeedMode(navMode);
+        }
+
+        public void SetNavMode(string navMode)
```

Comment:
We can add a shim method that accepts `StringValue` and calls this `string` method, eliminating another anonymous method.